src/hyperparameter_tuning.py

```diff
@@ -43,7 +43,7 @@ def fit(self, **kwargs):
             logger.info('training run {}'.format(i))
             logger.info('parameters: {}'.format(str(param_set)))
             transformer = self.TransformerClass(**param_set)
-            transformer.fit(**train_inputs)
+            transformer.fit(X=train_inputs['X'], y=train_inputs['y'])
```

Comment:
@Ninoko this makes it impossible to train with early stopping transformers like lgb